examples/web_view.html

```diff
@@ -1,50 +1,41 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Web Renderer Example</title>
-</head>
-
-<style>
-    html,
-    body {
-        color: white;
-        background-color: transparent;
-    }
-
-    canvas {
-        border: 2px solid red;
-        animation: anim infinite ease-in-out;
-        animation-duration: 3s;
-        width: 800px;
-        height: 600px;
-    }
-
-    @keyframes anim {
-
-        0%,
-        100% {
-            margin-left: 0;
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Web Renderer Example</title>
+    </head>
+
+    <style>
+        html,
+        body {
+            color: white;
+            background-color: transparent;
         }
 
-        50% {
-            margin-left: 250px;
+        canvas {
+            border: 2px solid red;
+            animation: anim infinite ease-in-out;
+            animation-duration: 3s;
+            width: 800px;
+            height: 600px;
         }
-    }
-</style>
 
-<body>
-    <h2>Frame embedding:</h2>
-    <canvas id="input_1"></canvas>
-</body>
+        @keyframes anim {
+            0%,
+            100% {
+                margin-left: 0;
+            }
 
-<script>
-    register_inputs(
-        "input_1"
-    );
-</script>
+            50% {
+                margin-left: 250px;
+            }
+        }
+    </style>
 
+    <body>
+        <h2>Frame embedding:</h2>
+        <canvas id="big_bunny_video"></canvas>
```

Comment:
Potential options
- [current] Use component_id directly for html id
- add some prefix e.g. for component id `big_bunny_video` html id could be `live_compositor:big_bunny_video`
  - a bit cleaner, scoped, harder to make collisions
  - can't be used to embed sth on a website you don't own
  
I agree with the current approach, but I'm assuming that it is possible to use this rendering on website you don't own. If there is already a reason why it would not be possible then I would prefer approach 2.

does this element need to be a canvas for all rendering options or just for chromium embeding?

Reply:
It's possible to use embedding on websites you don't own. I like the idea of using a namespace but I'm not sure if we want to limit the web renderer to websites designed specifically for the compositor.

For chromium embedding canvas element is required. Other embedding methods are not limited to canvases